public/javascripts/application.js

```diff
@@ -1,161 +1,142 @@
-function ShowHideContent() {
-  var self = this;
+/* global $ */
+/* global jQuery */
+/* global GOVUK */
```

Comment:
Note that an alternative way of avoiding this is doing:

```js
var $ = window.$
var GOVUK = window.GOVUK
```

Inside the code.

This works fine though. 👌